examples/flutter_gallery/ios/.gitignore

```diff
@@ -36,3 +36,5 @@ Icon?
 /Flutter/Flutter.framework
 /Flutter/Generated.xcconfig
 /ServiceDefinitions.json
+
+Pods/
```

Comment:
Added the missing newline.

The blank line was intentional; the file is already formatted into logical groupings, and I wanted to stay consistent with that.